command/git_push_test.go

```diff
@@ -151,20 +145,17 @@ func TestGitPush(t *testing.T) {
 			require.NoError(t, cmd.Run(ctx))
 
 			require.NoError(t, ioutil.WriteFile(path.Join(repoDir, "test3.txt"), []byte("test3"), 0644))
-			addToIndexCommands := []string{
+			toApplyCommands := []string{
 				`git rm test1.txt`,
 				`git add test3.txt`,
+				`git -c "user.name=baxterthehacker" -c "user.email=baxter@thehacker.com" commit -m "changes to push"`,
 			}
-			cmd = jpm.CreateCommand(ctx).Directory(repoDir).Append(addToIndexCommands...)
+			cmd = jpm.CreateCommand(ctx).Directory(repoDir).Append(toApplyCommands...)
 			require.NoError(t, cmd.Run(ctx))
 
 			params := pushParams{
-				directory:     repoDir,
-				authorName:    "baxterthehacker",
-				authorEmail:   "baxter@thehacker.com",
-				commitMessage: "testing 123",
-				branch:        "master",
-				skipCommit:    false,
+				directory: repoDir,
+				branch:    "master",
 			}
 			assert.NoError(t, c.pushPatch(ctx, logger, params))
```

Comment:
one could argue the entire `PushPatch` test is irrelevant now: since it's a dry run `c.pushPatch` just returns nil.

Reply:
I think the rest of the test is good, even though we don't need pushPatch (theoretically I could remove pushPatch but since it tests a lot of logic I'd rather just leave it: maybe dryRun won't do nothing forever!